python/cugraph/tests/utils.py

```diff
@@ -15,6 +15,48 @@
 import pandas as pd
 
 
+DATASETS_1 = ['../datasets/netscience.csv']
```

Comment:
Can't figure out the naming 2A 2B 4K etc.
Should we use self-documented names?

Reply:
My hope was that there would have been only a common dataset, that was not the case.  I'll updated the names, the numbers started off meaning how many items in the array - but then there were odd cases.   I'll have it redo today